common/util.go

```diff
@@ -63,10 +63,10 @@ const (
 	frontendClientRetryMaxAttempts     = 2
 
 	historyClientRetryInitialInterval = 50 * time.Millisecond
-	historyClientRetryMaxAttempts     = 2
+	historyClientRetryMaxAttempts     = 5
 
 	matchingClientRetryInitialInterval = 1000 * time.Millisecond
-	matchingClientRetryMaxAttempts     = 2
+	matchingClientRetryMaxAttempts     = 5
```

Comment:
Why doing this change?

Reply:
Because we don't retry on handler level anymore, it makes sense to increase number of attempts from client themselves. `2` is too small for overall retry attempts.